tests/slurm_tests/eval_test/llama_nemotron_super_49B_v1_5_evals/recipe.py

```diff
@@ -0,0 +1,269 @@
+
+    # Download the Llama-3_3-Nemotron-Super-49B-v1_5 model (to be evaluated) and Qwen2.5-32B-Instruct model (to be used as the judge model)
+    cmd = (
+        f"huggingface-cli download nvidia/Llama-3_3-Nemotron-Super-49B-v1_5 --local-dir {workspace}/Llama-3_3-Nemotron-Super-49B-v1_5 && "
+        f"huggingface-cli download Qwen/Qwen2.5-32B-Instruct --local-dir {workspace}/Qwen2.5-32B-Instruct"
+    )
+    run_cmd(
+        ctx=wrap_arguments(cmd),
+        cluster=cluster,
+        expname=f"{expname_prefix}-download-models",
+        log_dir=f"{workspace}/download-assets",
+    )
+
+    # Update config to support 128k
+    cmd = (
+        f'jq \'. + {{"rope_scaling": {{"type": "yarn", "factor": 4.0, "original_max_position_embeddings": 32768}}}}\' '
+        f"{workspace}/Qwen2.5-32B-Instruct/config.json > {workspace}/Qwen2.5-32B-Instruct/config_tmp.json && "
+        f"mv {workspace}/Qwen2.5-32B-Instruct/config_tmp.json {workspace}/Qwen2.5-32B-Instruct/config.json"
+    )
+
+    run_cmd(
+        ctx=wrap_arguments(cmd),
+        cluster=cluster,
+        expname=f"{expname_prefix}-patch-qwen-config",
+        log_dir=f"{workspace}/download-assets",
+        run_after=f"{expname_prefix}-download-models",
+        container="nemo",
+    )
+
+    # Prepare evaluation data - Local
+    cmd = "ns prepare_data gpqa mmlu-pro hle livecodebench scicode bfcl_v3 math-500 aime24 aime25"
+    subprocess.run(cmd, shell=True, check=True)
+
+    # Prepare ruler data - Mounted directory directly
+    ruler_data_cmd = f"""
+    ns prepare_data --cluster={cluster} \
+        --setup nemotron_super_128k \
+        --tokenizer_path nvidia/Llama-3_3-Nemotron-Super-49B-v1_5 \
+        --max_seq_length 131072 \
+        --data_dir {workspace}/ns-data \
+        --run_after {expname_prefix}-patch-qwen-config \
+        --expname {expname_prefix}-download-ruler-data
+    """
+    subprocess.run(ruler_data_cmd, shell=True, check=True)
+
+
+def get_inference_params(reasoning_mode: str) -> list:
+    """Get inference parameters as command line arguments."""
+    if reasoning_mode not in REASONING_MODES:
+        raise ValueError(f"Unknown reasoning mode: {reasoning_mode}")
+
+    config = REASONING_MODES[reasoning_mode]
+    return [
+        f"++inference.tokens_to_generate=65536",
+        f"++inference.temperature={config['temperature']}",
+        f"++inference.top_p={config['top_p']}",
+        "++skip_filled=True",
+        f"++system_message={config['system_message']}",
+    ]
+
+
+def run_evaluation(
+    workspace: str, cluster: str, expname_prefix: str, benchmark_name: str, reasoning_mode: str
+) -> None:
+    """Run a single benchmark evaluation."""
+    benchmark_config = BENCHMARK_CONFIGS[benchmark_name]
+    base_model = f"{workspace}/Llama-3_3-Nemotron-Super-49B-v1_5"
+
+    # Format output_dir with mode
+    output_dir = benchmark_config["output_dir"].format(mode=reasoning_mode)
+
+    # Build base command args
+    cmd_args = [
```

Comment:
using a big string with shell=True is easier to understand in my opinion. Similar to how we do for other tests